src/serac/physics/materials/tests/nonlinear_J2_material.cpp

```diff
@@ -172,10 +182,10 @@ TEST(FiniteDeformationNonlinearJ2Material, DerivativeCorrectness)
 
 TEST(FiniteDeformationNonlinearJ2Material, FrameIndifference)
 {
-  using Hardening = solid_mechanics::PowerLawHardening;
+  using Hardening = solid_mechanics::VoceHardening;
```

Comment:
This change is for test coverage, so that there is now one test using the Voce hardening rule.